### ISSUE TITLE
No switch of view modes

### ISSUE THREAD

**Issue by nwehrle** [chrisgrieser/shimmering-obsidian#9]
> now properly switches view mode, regardless of current view
> https://github.com/chrisgrieser/shimmering-obsidian/releases/tag/1.5.3

I do not think, it works. If I am in preview mode, it opens a 2nd panel in preview mode, too.

**Comment by chrisgrieser** [comment]
Hmm, I cannot reproduce the issue. Could you open the debug log (`cmd + D` in the workflow), run `odual`, and copy the debugging log?

Also, could you tell me what your default new pane mode is (under `Obsidian Settings ➞ Editor`), whether you have the "force view mode" plugin installed? 

Furthermore, when you are triggering `odual`, how many panes do you have open?

**Comment by nwehrle** [comment]
```
[21:19:28.935] STDERR: Shimmering Obsidian[Run Script] grep: /Users/nwehrle/Users/nwehrle/Library/Mobile Documents/iCloud~md~obsidian/Documents/MDVault/.obsidian/app.json: No such file or directory
grep: /Users/nwehrle/Users/nwehrle/Library/Mobile Documents/iCloud~md~obsidian/Documents/MDVault/.obsidian/workspace: No such file or directory
```

/Users/nwehrle is twice in the path

**Comment by nwehrle** [comment]
But the configuration in the alfred plugin looks OK.

**Comment by chrisgrieser** [comment]
🤔
Could you make a screenshot of your workflow configuration settings?

**Comment by nwehrle** [comment]
v1.53
<img width="796" alt="Screenshot 2021-08-31 at 08 04 43" src="https://user-images.githubusercontent.com/832234/131450703-05b59750-6399-4d75-8e4e-6141c2ef2fb1.png">

**Comment by nwehrle** [comment]
Maybe you can provide a debug build with extended logging? Can give a helping hand here.

**Comment by chrisgrieser** [comment]
No need, I already found out what the issue is. Have some day-job stuff to do, will release the fix later today :)

**Comment by nwehrle** [comment]
Day job .. I guess I know what you mean :)

**Comment by chrisgrieser** [comment]
The problem actually affected a few more commands :S

Nevertheless, fixed in [release 1.6.1](https://github.com/chrisgrieser/shimmering-obsidian/releases/tag/1.6.1)
